### ISSUE TITLE
The helper doesn't seem to work when there is a "homepage" setting in package.json

### ISSUE THREAD

**Issue by ruoshui-git** [MasterKale/django-cra-helper#11]
I had my "homepage": "/frontend" in package.json, and when doing "collectstatic --no-input", I got the following error:
```
$ python manage.py collectstatic --no-input
{'frontend/static/css/main.6e3fab79.chunk.css': 'main_css', 'frontend/static/js/main.7f0b0954.chunk.js': 'main_js', 'frontend/static/js/main.7f0b0954.chunk.js.map'
: 'main_js_map', 'frontend/static/js/runtime-main.e2bed40b.js': 'runtime_main_js', 'frontend/static/js/runtime-main.e2bed40b.js.map': 'runtime_main_js_map', 'front
end/static/css/2.11a187e6.chunk.css': 'static_css_2_11a187e6_chunk_css', 'frontend/static/js/2.ec811320.chunk.js': 'static_js_2_ec811320_chunk_js', 'frontend/stati
c/js/2.ec811320.chunk.js.map': 'static_js_2_ec811320_chunk_js_map', 'frontend/index.html': 'index_html', 'frontend/precache-manifest.faa980a089f14cfd9dcd994ddf3709
db.js': 'precache_manifest_faa980a089f14cfd9dcd994ddf3709db_js', 'frontend/service-worker.js': 'service_worker_js', 'frontend/static/css/2.11a187e6.chunk.css.map':
 'static_css_2_11a187e6_chunk_css_map', 'frontend/static/css/main.6e3fab79.chunk.css.map': 'static_css_main_6e3fab79_chunk_css_map', 'frontend/static/js/2.ec811320
.chunk.js.LICENSE.txt': 'static_js_2_ec811320_chunk_js_LICENSE_txt'}
Traceback (most recent call last):
  File "manage.py", line 21, in <module>
    main()
  File "manage.py", line 17, in main
    execute_from_command_line(sys.argv)
  File "C:\Users\known\Documents\code\manager\venv\lib\site-packages\django\core\management\__init__.py", line 401, in execute_from_command_line
    utility.execute()
  File "C:\Users\known\Documents\code\manager\venv\lib\site-packages\django\core\management\__init__.py", line 377, in execute
    django.setup()
  File "C:\Users\known\Documents\code\manager\venv\lib\site-packages\django\__init__.py", line 24, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "C:\Users\known\Documents\code\manager\venv\lib\site-packages\django\apps\registry.py", line 91, in populate
    app_config = AppConfig.create(entry)
  File "C:\Users\known\Documents\code\manager\venv\lib\site-packages\django\apps\config.py", line 90, in create
    module = import_module(entry)
  File "C:\Python38\lib\importlib\__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "C:\Users\known\Documents\code\manager\venv\lib\site-packages\cra_helper\__init__.py", line 33, in <module>
    STATIC_ASSET_MANIFEST = generate_manifest(CRA_URL, CRA_FS_APP_DIR)
  File "C:\Users\known\Documents\code\manager\venv\lib\site-packages\cra_helper\asset_manifest.py", line 114, in generate_manifest
    rel_static_path = manifest[mapped_manifest_items[path]]
KeyError: 'static/js/runtime-main.e2bed40b.js'
```

(I modified your package to print out the "mapped_manifest_items" after line 112 in asset_manifest.py, which results in the dict in the beginning)

Not only does collectstatic not work, django itself seems to be broken, since I can't even run "check".

Then I removed the "homepage" setting from package.json, I got (the second time, actually, so no new files were copied, but it was successful)
```
$ python manage.py collectstatic --no-input
{'static/css/main.6e3fab79.chunk.css': 'main_css', 'static/js/main.c67dbaad.chunk.js': 'main_js', 'static/js/main.c67dbaad.chunk.js.map': 'main_js_map', 'static/js
/runtime-main.1f1f19a3.js': 'runtime_main_js', 'static/js/runtime-main.1f1f19a3.js.map': 'runtime_main_js_map', 'static/css/2.11a187e6.chunk.css': 'static_css_2_11
a187e6_chunk_css', 'static/js/2.9d1ff8ae.chunk.js': 'static_js_2_9d1ff8ae_chunk_js', 'static/js/2.9d1ff8ae.chunk.js.map': 'static_js_2_9d1ff8ae_chunk_js_map', 'ind
ex.html': 'index_html', 'precache-manifest.ccfb517866d1102d0142a5fc91597f11.js': 'precache_manifest_ccfb517866d1102d0142a5fc91597f11_js', 'service-worker.js': 'ser
vice_worker_js', 'static/css/2.11a187e6.chunk.css.map': 'static_css_2_11a187e6_chunk_css_map', 'static/css/main.6e3fab79.chunk.css.map': 'static_css_main_6e3fab79_
chunk_css_map', 'static/js/2.9d1ff8ae.chunk.js.LICENSE.txt': 'static_js_2_9d1ff8ae_chunk_js_LICENSE_txt'}

0 static files copied to 'C:\Users\known\Documents\code\manager\staticfiles', 143 unmodified, 325 post-processed.
```

**Comment by MasterKale** [comment]
@ruoshui-git Thank you for the detailed information, that helps a lot. I'm unfamiliar with CRA's support for `homepage` in package.json, but I think I see why my code is choking on it.

Can I trouble you for two copies of your project's raw **asset-manifest.json** file?

- The "broken" one, with `homepage` set
- The "working" one, when `homepage` is removed from package.json

**Comment by MasterKale** [comment]
@ruoshui-git This should be fixed with **v1.2.2** I just released:

https://pypi.org/project/django-cra-helper/1.2.2

**Comment by ruoshui-git** [comment]
> @ruoshui-git Thank you for the detailed information, that helps a lot. I'm unfamiliar with CRA's support for `homepage` in package.json, but I think I see why my code is choking on it.
> 
> Can I trouble you for two copies of your project's raw **asset-manifest.json** file?
> 
> * The "broken" one, with `homepage` set
> * The "working" one, when `homepage` is removed from package.json

Hi! I'm sorry for the late response. But since you've already fixed the issue, I'm guessing that you don't need these anymore?
Unfortunately I've separated my frontend project to another directory entirely. But I can try to reproduce this if you'd like.
